decentraland/explorer
# fix: "Browser not supported" error screen missing

# What does this PR fix? 
## Fixes https://github.com/decentraland/unity-renderer/pull/708
This error screen is not being showed when trying to run Decentraland with Safari.

![Screen Shot 2021-06-28 at 14.35.30.png](https://images.zenhubusercontent.com/5d91f7e9df37660001a11578/641ec26d-2b6b-4265-a79f-6fcb09e3b6a1)

The words "Chrome" and "Firefox" are links:
- **Chrome**: https://www.google.com/chrome/?brand=BNSD&gclid=Cj0KCQjw5uWGBhCTARIsAL70sLJsOBpT9ZDBOEtFx49Ya-nc0iCR50klEV8iLNoOdkeVtVly8x9_gXUaAhSdEALw_wcB&gclsrc=aw.ds
- **Firefox**: https://www.mozilla.org/en-US/firefox/new/

### How to test it?
1. Open Safari browser.
2. Go to https://play.decentraland.zone/branch/fix/Browser-not-supported-error-screen-missing/index.html?ENV=org
3. Notice the error screen is showed.
4. Close Safari browser and open Chrome or Firefox.
5. Go to https://play.decentraland.zone/branch/fix/Browser-not-supported-error-screen-missing/index.html?ENV=org
6. Notice the world loads as usual.
